src/components/vr_cooperation/src/register_service_request.cc

```diff
@@ -30,7 +30,9 @@
  POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include "functional_module/function_ids.h"
 #include "vr_cooperation/commands/register_service_request.h"
+#include "vr_cooperation/vr_module_constants.h"
 #include "utils/logger.h"
```

Comment:
each section of includes should be sorted
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes